### DISCUSSION TITLE
Range

### DISCUSSION THREAD

**Discussion by danielducu** [PricelessToolkit/MailBoxGuard#47] [General]
Ok, so I started testing everything. I have SF set to 12 and the gateway is placed in the middle of my apartment. Walls are drywall with metal frame. I get a -100dbm 5 floors bellow and 5 floors up but when I get out ... 200 meters away from my block packets don't make it anymore. Is this close to normal for this setup or maybe I've had higher expectations... maybe the antenna provided with the lilygo module is bad ? Or maybe the helicoidal copper antenna that comes with the Lora module is crap?

**Comment by PricelessToolkit** [comment]
-100dbm is good it can go up to -130dbm

1. As mentioned in the video, the gateway's antenna lacks quality. For optimal performance, consider replacing your gateway antenna.
2. Remember, antennas have polarization. Align the gateway and sensor antennas vertically.

**Reply by danielducu** [reply]
Ok man, overall im very happy with the sensor. It's amazing I can place it inside my full metal mailbox, 5 floors bellow and get a good working signal ... it's crazy. Thanks a lot for your hard work.
